hibernate-reactive-core/src/test/java/org/hibernate/reactive/BasicTypesAndCallbacksTest.java

```diff
@@ -25,6 +31,10 @@ protected Configuration constructConfiguration() {
 
 	@Test
 	public void testBasicTypes(TestContext context) {
+		// TODO @AGG
+		// The DB2 driver does not yet support a few types (BigDecimal, BigInteger, LocalTime)
+		// so we need to keep a separate copy around for testing DB2 (DB2BasicTest)
+		assumeTrue( DatabaseConfiguration.dbType() != DBType.DB2 );
```

Comment:
That would also be a good way to do it. Its still good to have the all-in-one test though because I have seen 1-2 bugs in the client drivers where operating on certain combinations of column types causes issues.

Reply:
ok, I thought about that and don't expect to be a problem but let's be safe :)